python/ray/data/_internal/util.py

```diff
@@ -421,3 +425,38 @@ def capitalize(s: str):
         Capitalized string with no underscores.
     """
     return "".join(capfirst(x) for x in s.split("_"))
+
+
+def _df_to_block(df: "pandas.DataFrame") -> "Block[ArrowRow]":
```

Comment:
let's remove `_` as they are utility methods, and I think a explicit naming would be helpful:

```
def pandas_df_to_arrow_block()
```